docs/dataset/object-detection.md

```diff
@@ -0,0 +1,334 @@
+---
+icon: kolena/detection-20
+---
+
+# :kolena-detection-20: How to Test Object Detection Models
+
+This guide outlines how to format your object detection data on Kolena, and walks through the steps to easily
+test object detection models.
+
+## Overview of Object Detection
+
+<div class="grid" markdown>
+<div markdown>
+Object detection is a widely applied computer vision task that aims to locate and identify objects
+within an image or video, typically with bounding boxes.
+Object detection has a wide range of applications such as self-driving cars, facial recognition,
+video surveillance, medical imaging, robotics, and more. Therefore, it is crucial to evaluate object detection
+models to understand their performance and applicability in any real-world situation.
+</div>
+<figure markdown>
+![COCO Multiclass OD](../assets/images/task-od-multiclass.png)
+<figcaption markdown>Vehicle detection on an [MS COCO](https://paperswithcode.com/dataset/coco) image</figcaption>
+</figure>
+</div>
+
+??? info "Examples of Object Detection Datasets"
+
+    Object detection models require large amounts of annotated data to learn about objects of interest.
+    Some commonly used
+    datasets are: [MS COCO](https://paperswithcode.com/dataset/coco),
+    [Pascal VOC](https://paperswithcode.com/dataset/pascal-voc),
+    [Open Images](https://paperswithcode.com/dataset/open-images-v7),
+    and [ImageNet](https://paperswithcode.com/dataset/imagenet).
+
+## Object Detection Model Evaluation
```

Comment:
Probably best to not have OD in every header, given that the page is for OD already
```suggestion
## Model Evaluation
```